olclient/schemata/import.schema.json

```diff
@@ -11,13 +11,29 @@
   ],
   "additionalProperties": false,
   "properties": {
-    "title":    { "type": "string" },
-    "subtitle": { "type": "string" },
-    "source_records":   { "$ref": "shared_definitions.json#/string_array" },
-    "publishers":   { "$ref": "shared_definitions.json#/string_array" },
+    "title": {
+      "type": "string"
+    },
```

Comment:
Don't reformat the entire file, that makes the diff complicated to read. Just fix the formatting on your specific changes :+1:

Reply:
My bad, I should have stuck to the instructions.